rook/scripts/test-rook.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+LOGGING=${2:-false}
+SUCCESSES=0
+FAILURES=0
+DEBUG_OUTPUT=("")
+CONFIG_FILE="${CK8S_CONFIG_PATH}/rook/values.yaml"
+here="$(dirname "$(readlink -f "$0")")"
+# shellcheck source=rook/scripts/funcs.sh
+source "${here}/funcs.sh"
+
+function test_rook_help() {
+  printf "%s\n" "[Usage]: test-rook.sh <sc|wc|both> [--logging-enabled]"
+  exit "${1:-0}"
+}
+
+function test_rook() {
+  case "${1}" in
+  sc)
+    clusters="sc"
+    ;;
+  wc)
+    clusters="wc"
+    ;;
+  both)
+    clusters="sc wc"
+    ;;
+  esac
+
+  for cluster in $clusters; do
+    export KUBECONFIG="${CK8S_CONFIG_PATH}/.state/kube_config_${cluster}.yaml"
+    DEPLOYMENTS=$(kubectl -n rook-ceph get deployments.apps -o=jsonpath='{.items[*].metadata.name}' |  xargs -n 1)
```

Comment:
Maybe we should change this into the shellcheck approved way https://github.com/koalaman/shellcheck/wiki/SC2207

And then also change the for loops to

```bash
for deployment in "${DEPLOYMENTS[@]}"; do
...
```